scripts/seed-database/container.sh

```diff
@@ -31,6 +31,7 @@ tables=(
   # Please comment in if working on a feature and you require example data locally
   # You will need to manually grant select permissions to the github_actions on production, and update main.sql
   # feedback
+  # flow_comments
```

Comment:
NOT including `flow_comments` in the nightly data sync, but did mock them up here similar to `feedback` so easy to "toggle on" for local dev if desired which I found helpful for building out the modal